### ISSUE TITLE
When will menuconfig be updated

### ISSUE THREAD

**Issue by versamodule** [platformio/platform-espressif32#899]
When it comes to the ESP32 platform, menuconfig is well…To DOS like. When others have made it far more GUI like. When is PlatformIO going to put more effort into doing the same ?
Espressif’s IDF extension uses VSCode, so I can see no reason why PlatformIO cannot do the same when it comes to menuconfig. Even Eclipse using ESP-IDF has the same GUI interface.

Here is PlatformIO’s menuconfig.

![1](https://user-images.githubusercontent.com/6456652/188237765-0077b1f6-c4f5-4561-b796-4cd5e3c1da5d.png)

This is Espressif’s IDF extension, and Eclipse is similar. 

![2](https://user-images.githubusercontent.com/6456652/188237853-2cadb754-7508-449b-a9c0-81e7d238ae3a.png)

**Comment by Jason2866** [comment]
Yes, the Espressif VSC extension does look nice. Imho it is cluttered. The old style menuconfig is not looking so nice but is structured.

**Comment by versamodule** [comment]
"Imho it is cluttered" you really thing so? 
I am shocked that someone would think that. It's all well organized, and it's also structured.  If you are saying that because of the photo I would suggest trying it. Can't see how anyone would want to use the current method compared to the GUI way.  Not to mention you have to use the J and K keys to move up and down.
